plat/drivers/gic/gic-v3.c

```diff
@@ -499,7 +499,7 @@ static void gicv3_init_dist(void)
 	uk_pr_info("GICv3 distributor initialized.\n");
 }
 
-static void gicv3_handle_irq(void)
+static void gicv3_handle_irq(struct __regs *regs)
```

Comment:
Don't forget to add `__unused` attributes if necessary.

Reply:
Is this a more general comment? `regs` is used in the function.

Reply:
Ah, yes. My bad 👍🏻